packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -37,12 +41,6 @@ export interface TextInputProps extends CommonFieldProps {
    */
   type?: "text" | "email" | "url" | "tel" | "search" | "password";
 
-  /**
-   * Callback to provide a reference to the underlying input element
-   * @param {Object} instance the input element or null
-   */
-  inputRef?: RefCallback<HTMLInputElement>;
-
```

Comment:
using a forwarded ref now - added a breaking change commit title to call this out